server/documents/elements/emoji.html.eco

```diff
@@ -0,0 +1,115 @@
+---
+layout      : 'default'
+css         : 'emoji'
+standalone  : true
+
+element     : 'emoji'
+elementType : 'element'
+
+title       : 'Emoji'
+description : 'An icon is a glyph used to represent an emotinal state'
+type        : 'UI Element'
+
+themes      : ['Default']
+---
+
+<%- @partial('header', { tabs: { emoji: 'Emojis', definition: 'Definition' } }) %>
+
+<script src="/javascript/emoji_strategy.js"></script>
+<script src="/javascript/emoji.js"></script>
+
+<div class="main ui container">
+
+  <div class="ui active tab" data-tab="emoji">
+
+    <emojiSearch class="ui right aligned search">
+      <div class="ui icon input">
+        <input class="prompt" type="text" placeholder="Search emojis...">
+        <i class="search icon"></i>
+      </div>
+      <div class="results"></div>
+    </emojiSearch>
+
+    <h2 class="ui header">Emoji Set</h2>
+    <p>An emoji set contains all <a href="https://unicode.org/emoji/charts/full-emoji-list.html">Unicode 12.1 Emoji Characters</a></p>
+    <div class="ui ignored warning message">
+      Emojis serve a very similar function to text in a page. In Fomantic emojis receive a special tag <code>&lt;em&gt;</code> which allow for an abbreviated markup when sitting along-side text.
+        <br>
+        Because of the complexity and possible interferance with existing fomantic classnames, emoji names have to be provided via the <code>data-emoji</code> attribute instead of usual classnaming.
+    </div>
+    <div class="ui ignored message">
+        By default all emoji images are linked to CDN and are <b>not part of FUI itself</b>.
```

Comment:
Should read "By default all emoji images are linked to a CDN and are not part of Fomantic itself"

Reply:
Done in fafc1edc76e8f5256dc395224881ee7bccd48efd 🙂